packages/core/src/redocly/index.ts

```diff
@@ -68,11 +88,11 @@ export class RedoclyClient {
       process.exit(1);
     }
 
-    this.accessToken = accessToken;
+    this.accessTokens = { [this.region]: accessToken };
     const credentials = {
-      token: accessToken,
+      ...(existsSync(credentialsPath) && JSON.parse(readFileSync(credentialsPath, 'utf-8'))),
```

Comment:
Move this into a separate function:

`existsSync(credentialsPath) && JSON.parse(readFileSync(credentialsPath, 'utf-8')`.

It can return `{}` by default (if file doesn't exist) which will simplify the code.